.github/dependabot.yml

```diff
@@ -3,53 +3,80 @@ updates:
   - package-ecosystem: docker
     directory: /build/gator
     schedule:
-      interval: daily
+      interval: weekly
     commit-message:
       prefix: "chore"
       
   - package-ecosystem: docker
     directory: /build/gomplate
     schedule:
-      interval: daily
+      interval: weekly
     commit-message:
       prefix: "chore"
       
   - package-ecosystem: gomod
     directory: /scripts/artifacthub
     schedule:
-      interval: daily
+      interval: weekly
     commit-message:
       prefix: "chore"
     ignore:
       - dependency-name: "*"
         update-types:
         - "version-update:semver-major"
+        - "version-update:semver-minor"
+      
+  - package-ecosystem: "gomod"
+    directory: "/scripts/validate"
+    schedule:
+      interval: weekly
+    commit-message:
+      prefix: "chore"
+    ignore:
+      - dependency-name: "*"
+        update-types:
+        - "version-update:semver-major"
+        - "version-update:semver-minor"
 
   - package-ecosystem: gomod
     directory: /scripts/require-sync
     schedule:
-      interval: daily
+      interval: weekly
     commit-message:
       prefix: "chore"
     ignore:
       - dependency-name: "*"
         update-types:
         - "version-update:semver-major"
+        - "version-update:semver-minor"
 
   - package-ecosystem: gomod
     directory: /scripts/website
     schedule:
-      interval: daily
+      interval: weekly
     commit-message:
       prefix: "chore"
     ignore:
       - dependency-name: "*"
         update-types:
         - "version-update:semver-major"
+        - "version-update:semver-minor"
 
   - package-ecosystem: npm
     directory: /website
     schedule:
-      interval: daily
+      interval: weekly
+    commit-message:
+      prefix: "chore"
+    ignore:
+      - dependency-name: "*"
+        update-types:
+        - "version-update:semver-major"
+        - "version-update:semver-minor"   
+
+  - package-ecosystem: github-actions
+    directory: /
```

Comment:
```suggestion
    directory: /.github
```

Reply:
this is interesting, we only have `/` in GK but definately get GHA updates

Reply:
i am guessing `.github/workflows` is built-in since actions must live there

Reply:
Yeah, we also have `/` on other repos which works just fine. I'll keep this as is since it's tested.